src/Native/System.Private.CoreLib.Native/pal_datetime.cpp

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include <stdlib.h>
+#include <sys/time.h>
+
+/*
+   To 1970:
+   Both epochs are Gregorian. 1970 - 1601 = 369. Assuming a leap
+   year every four years, 369 / 4 = 92. However, 1700, 1800, and 1900
+   were NOT leap years, so 89 leap years, 280 non-leap years.
+   89 * 366 + 280 * 365 = 134744 days between epochs. Of course
+   60 * 60 * 24 = 86400 seconds per day, so 134744 * 86400 =
+   11644473600 = SECS_BETWEEN_1601_AND_1970_EPOCHS.
+*/
+
+static const __int64 SECS_BETWEEN_1601_AND_1970_EPOCHS = 11644473600LL;
```

Comment:
We should be using the standard C++ types here: int64_t, etc.

__int64 is Windowsism.